src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/OasysService.kt

```diff
@@ -122,6 +125,20 @@ class OasysService(val oasysApiClient: OasysApiClient) {
     return oasysPsychiatric.toModel()
   }
 
+  fun getLearningNeeds(prisonNumber: String): LearningNeeds? {
+    val assessmentId = getAssessmentId(prisonNumber)
+    var learningNeeds = LearningNeeds()
+    if (assessmentId != null) {
+      val oasysLearning = getLearning(assessmentId)
+      val oasysAccommodation = getAccommodation(assessmentId)
+      learningNeeds = learningNeeds.toModel(oasysAccommodation, oasysLearning)
```

Comment:
We can remove the additional LearningNeeds() and inline it?

`return LearningNeeds().toModel(oasysAccommodation, oasysLearning)`